src/frontend/src/pages/WorkPackageTemplateForm/WorkPackageTemplateFormView.tsx

```diff
@@ -0,0 +1,169 @@
+import React from 'react';
+import { Controller, useFieldArray, useForm } from 'react-hook-form';
+import { yupResolver } from '@hookform/resolvers/yup';
+import { Box, TextField, Autocomplete, Typography, Stack, FormControl } from '@mui/material';
+import NERSuccessButton from '../../components/NERSuccessButton';
+import PageLayout from '../../components/PageLayout';
+import { useToast } from '../../hooks/toasts.hooks';
+import { WorkPackageTemplateApiInputs } from '../../apis/work-packages.api';
+import { DescriptionBulletPreview, WorkPackageStage } from 'shared';
+import { ObjectSchema } from 'yup';
+import { useHistory } from 'react-router-dom';
+import { routes } from '../../utils/routes';
+import DescriptionBulletsEditView from '../../components/DescriptionBulletEditView';
+import { NERButton } from '../../components/NERButton';
+import WorkPackageTemplateFormDetails from './WorkPackageTemplateFormDetails';
+
+interface WorkPackageTemplateFormViewProps {
+  exitActiveMode: () => void;
+  workPackageTemplateMutateAsync: (data: WorkPackageTemplateApiInputs) => void;
+  defaultValues?: WorkPackageTemplateFormViewPayload;
+  blockedByOptions: { id: string; label: string }[];
+  schema: ObjectSchema<any>;
+}
+
+export interface WorkPackageTemplateFormViewPayload {
+  workPackageName?: string;
+  templateName: string;
+  templateNotes: string;
+  workPackageTemplateId: string;
+  duration?: number;
+  stage: string;
+  blockedBy: { id: string; label: string }[];
+  descriptionBullets: DescriptionBulletPreview[];
+}
+
+const WorkPackageTemplateFormView: React.FC<WorkPackageTemplateFormViewProps> = ({
+  exitActiveMode,
+  workPackageTemplateMutateAsync,
+  defaultValues,
+  blockedByOptions,
+  schema
+}) => {
+  const toast = useToast();
+  const {
+    register,
+    handleSubmit,
+    control,
+    watch,
+    formState: { errors }
+  } = useForm<WorkPackageTemplateFormViewPayload>({
+    resolver: yupResolver(schema),
+    defaultValues: {
+      workPackageName: defaultValues?.workPackageName ?? '',
+      templateName: defaultValues?.templateName ?? '',
+      templateNotes: defaultValues?.templateNotes ?? '',
+      workPackageTemplateId: defaultValues?.workPackageTemplateId ?? '',
+      duration: defaultValues?.duration ?? 0,
```

Comment:
Duration is actually optional so we dont want this 0, (will set 0 as the duration when we don't want any duration set)